.circleci/config.yml

```diff
@@ -130,7 +130,7 @@ jobs:
             # we need to re-collect static as docker-compose mounts the checkout directory over what we have in the image
             chmod 777 . # make sure we have permission to write the files in our volume
             # run the actual tests
-            make test
+            make leantest
```

Comment:
I'm not 100% sure what this leantest stuff is, is it about not running redundant checks in CI? Probably easier addressed if we move to a seperate PR.

Reply:
"leantest" is what we discussed a while ago that disables some pytest plugins in CI: https://github.com/iodide-project/iodide/pull/2219/files#diff-6b435363a6346c7164cb2f6e347ec389 Basically if you run `pytest --lean` it'll disable pytest-flake8, pytest-isort and pytest-coverage. Since you've said in our discussion that you dislike having all that as part of the CI run and since there is already an overlap with the lint scripts I added that ability. That means running `make test` during development will actually use flake8, isort and coverage.

Reply:
awesome, makes sense!